roles/openshift-applier/README.md

```diff
@@ -43,10 +43,12 @@ The `tags` definition is a list of tags that will be processed if the `filter_ta
 
 #### Sourcing a directory with files
 
-You can source a directory composed of static files (without parameters) using `content_dir` instead of defining each file individually. That would look like this:
+You can source a directory composed of static files (without parameters) using `files` instead of defining each file individually. That would look like this:
```

Comment:
Can we add a `Note:` below here about `files` replaces `content_dir`, which used to exist ...